tests/conditionalVault.ts

```diff
@@ -907,11 +928,10 @@ async function initializeDepositSlip(
 
 async function generateRandomVault(
   vaultProgram: VaultProgram,
+  payer: Keypair,
+  banksClient: BanksClient,
   settlementAuthority: Keypair = anchor.web3.Keypair.generate()
-): [PublicKey, Keypair, Keypair] {
-  const banksClient = vaultProgram.provider.connection.banksClient;
```

Comment:
did this give a type error or something?

Reply:
Yes, there is no `banksClient` in `web3.Connection`